api/src/producer/controller/__tests__/producerController.ts

```diff
@@ -0,0 +1,157 @@
+import { faker } from '@faker-js/faker'
+
+import request from 'supertest'
+import { Express } from 'express-serve-static-core'
+
+import db from '../../../../utils/db'
+import { createServer } from '../../../../utils/server'
+import Producer from '../../model/producer'
+
+let server: Express
+
+const baseUrl = '/api/producers/v1'
+
+beforeAll(async () => {
```

Comment:
Zazwyczaj zamykam wszystkie testy z jednego pliku w nadrzędnego `describe`. Wolę mieć to potem w raporcie zgrupowane w jednym miejscu. 
Chyba na nic innego to nie powinno mieć wpływu, ale tak się sam zacząłem zastanawiać, czy aby czasem w jakieś implementacji test runnera taki `beforeAll` nie będzie się odpalał dla testów z innego pliku... 🤔 😅